src/main/kotlin/com/tradeshift/blayze/features/Text.kt

```diff
@@ -11,7 +11,7 @@ import com.tradeshift.blayze.dto.Outcome
  *
  * The pre-processing replaces all non-letters and non-numbers with spaces, lowercases, splits on spaces and finally removes english stopwords.
  */
-class Text(private val delegate: Multinomial = Multinomial()) : Feature<Text, FeatureValue> {
+class Text(private val delegate: Multinomial = Multinomial()) : Feature<Text, FeatureValue, Multinomial.PseudoCount, Multinomial.IncludeFeatureProbability> {
```

Comment:
Consider whether some of the hardcoded values in the text pre-processing should be Params. They would only make sense if it makes sense changing them at "runtime".